website/docs/advanced/strategy-constraints.md

```diff
@@ -4,51 +4,70 @@ title: Strategy Constraints
 ---
 
 <div class="alert alert--info" role="alert">
-  Strategy constraints part of Unleash Pro and Enterprise.
+  Strategy constraints are part of Unleash Pro and Enterprise.
 </div>
-<br />
+<BR />
```

Comment:
Why uppercase - Something to do with how docusaurus parses our markdown?

Reply:
Nope, not at all. I was surprised to see this too, must've been a misplaced editor command. I'll fix it though 🙋🏼

Reply:
Fixed in aa80cba2221588d75b820db17bf2884e2d752eab